src/core/utils_test.go

```diff
@@ -130,6 +131,18 @@ func TestLookPathColons(t *testing.T) {
 	assert.Equal(t, "bash", info.Name())
 }
 
+func TestIsSameFile(t *testing.T) {
+	err := ioutil.WriteFile("issamefile1.txt", []byte("hello"), 0644)
```

Comment:
plz will clean them for us :) the test runs in a temporary directory that it cleans up after.